WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js

```diff
@@ -1721,6 +1731,23 @@ function(by, val) {
     } else {
         this[ZaAccount.A2_autodisplayname] = "FALSE";
     }
+    let arr=["domainAdminZimletRights","adminLoginCalendarResourceAs","domainAdminConsoleRights"];
```

Comment:
Don't use ES6 Features.

Reply:
moved the changes to https://github.com/Zimbra/zm-delegated-admin-zimlet/pull/8